src/game/Tactical/Turn_Based_Input.cc

```diff
@@ -2413,12 +2395,6 @@ void GetKeyboardInput(UIEventKind* const puiNewEvent)
 			}
 			else if ( InputEvent.usKeyState & CTRL_DOWN )
       {
```

Comment:
Removal won't be trivial to review, or implement.  Cutting the block puts more keystrokes in the else block which appears three lines below (due to poor formatting and poor newline / brace style choices, it appears to be an else block for a different else statement).

I don't mind cleaning up the flock-of-seagulls nesting smell, but I haven't determined if I'll be doing that in this branch.  Currently I'm leaning against it, but if it's the only item left, please consider merging without repair in this case.

The empty block is the safest, even if it is less clear.